### PR TITLE
copy license file to chart.

### PR DESCRIPTION
**Pull request by morvencao**
Following the best practice of helm chart: https://docs.helm.sh/developing_charts/
Let's copy `LICENSE` file to chart directory.

### CODE REVIEW

File: install/kubernetes/helm/istio-remote/LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "{}"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2016 Istio Authors
```
[the remark this answers is not in the document]
Comment (morvencao) [reply install/kubernetes/helm/istio-remote/LICENSE]: Done in latest commit, and also update this in https://github.com/istio/istio/blob/master/LICENSE#L190

File: LICENSE [hunk 2]
```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2016 Istio Authors
+   Copyright 2018 Istio Authors
```
Comment (gyliu513) [inline LICENSE]: You do not need to update this license but only the new license

Comment (morvencao) [reply LICENSE]: done

**Pull request merged**